source/element-parameters.md

```diff
@@ -248,7 +253,131 @@ particle point and the `center` point.
+are needed to specify a multipole. 
+However, it is sometimes convenient to use all three. 
+For example, {Math}`B_n` and {math}`B_s` with {math}`N=0` can be used to describe 
+an element with independent horizontal and vertical steerings can {math}`T` can be used
+to represent rotational errors.
+
+The components of `BMultipoleP` for specifying a multipolar field of order `N` is:
+```{code} yaml
+tiltN     - Tilt
+BnN       - Normal component 
+BsN       - Skew component
+```
+
+Alternatively, the field components can be specified using normalized values
+```{code} yaml
+KnN       - Normalized normal component 
+KsN       - Normalized skew component
+```
+where the conversion between field and normalized components is:
+```{math}
+  (K_n, K_s) = \frac{q}{P_0} \, (B_n, B_s)
+```
+with {math}`q` being the charge of the reference particle and {math}`P_0` being the 
+reference momentum.
+
+Furthermore, the field and normalized values can be given in terms of the integrated strength.
+Integrated values are specified with the letter 
+`L` appended at the end of the name. Example:
+```{code} yaml
+BMultipoleP:
+  tilt7: 0.7        - Tilt of 7th order multiple
+  Bn3: 27.3         - Normal multipole component of order 3.
+  Bn3L: 3.47e1      - Length integrated normal multipole component of order 3.
+```
+The length integrated values are related to the non-integrated values via
+```{math}
+  (B_n, B_s, K_n, K_s) = L \, (B_n, B_s, K_n, K_s)
+```
+where `L` is the length of the element.
+
+For a given element, when specifying a multipole of a given order, 
+the two strength components must be of the same type.
+That both must be length integrated or not, and both must be the field or normalized components.
+However, the multipole components of different order do not have to be of the same type.
+
+When multipoles are specified for a `Bend` element, the calculation of the field is
+complicated by the curvilinear coordinate system.
+the `geometry` component switch can be used to specify how to calculate fields. 
+Possible settings for this component are:
+```{code} yaml
+vertically_pure
+horizontally_pure
+entrance_tangent
+exit_tangent
+chord_tangent
+```
+The `entrance_tangent` setting is used when the reference coordinate system for the multipoles
+is the straight line tangent to the entrance coordinates of the bend. 
+Similarly, the `exit_tangent` setting is used when the  reference coordinates are the 
+straight line tangent to the exit coordinates of the bend and the `chord_tangent` setting is used
+when the reference coordinates are the straight line connecting the entrance point to the
+exit point. In all these three cases the coordinate system is a straight line so Eq. [](#bbmult)
+is valid.
+
+For `geometry` set to `vertically_pure` or `horizontally_pure` the referece coordinate system
+for the multipoles is the circular arc of the bend. This is discussed in detail
+in [](#c:multipole.bend).
```

Comment:
Similarly to previous paragraph, "the referece coordinate system for the multipoles is the circular arc of the bend" is unclear. Probably better to discuss.

Also, typo in "referece".

Reply:
Fixed.